docs/my-website/docs/providers/mistral.md

```diff
@@ -203,6 +205,110 @@ assert isinstance(
 )
 ```
 
+## Reasoning Capabilities (Magistral Models)
+
+Mistral's Magistral models support advanced reasoning capabilities that allow the model to think step-by-step before providing answers. LiteLLM provides seamless integration with these reasoning features through OpenAI-compatible parameters.
+
+### Supported Magistral Models
+
+| Model Name     | Function Call                                                |
+|----------------|--------------------------------------------------------------|
+| Magistral Small  | `completion(model="mistral/magistral-small-2506", messages)` |
+| Magistral Medium | `completion(model="mistral/magistral-medium-2506", messages)`|
+
+### Using Reasoning Effort
+
+The `reasoning_effort` parameter controls how much effort the model puts into reasoning. When used with magistral models.
+
+```python
```

Comment:
nit: can you add showLineNumbers for all code snippets on docs. Trying to get more docs like this as it looks polished